assets/app/data/crowbar/services/crowbar.factory.js

```diff
@@ -41,7 +41,7 @@
 
```

Comment:
(for some reason I cannot add this comment on line 12)
This method (`getRepositoriesChecks `) needs to be moved from crowbar factory to upgrade factory, as they're intended to be a reflection of the API entities.